config/Dockerfiles/release-notes.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/python
+
+from github import Github
+import sys
+
+REPO="CentOS-PaaS-SIG/linchpin"
```

Comment:
Is there any reason for hardcoding repo name Instead of passing it as runtime parameter?

Reply:
We can have fewer runtime parameters this way.  More importantly, it's unlikely that the repo will ever change so there's really no harm in including as a constant, unlike the version and the key which are variable and secret respectively